src/sentry/incidents/endpoints/organization_incident_details.py

```diff
@@ -56,8 +55,6 @@ def put(self, request: Request, organization, incident) -> Response:
                 incident = update_incident_status(
                     incident=incident,
                     status=result["status"],
-                    user=serialize_generic_user(request.user),
-                    comment=result.get("comment"),
```

Comment:
This is the only call to `update_incident_status` that passes `user` and `comment` so I checked the API access logs to confirm the only usage is by customers who are probably using it to close incidents. I also ensured that we haven't sent the incident activity emails just in case a customer was for some reason using that part of it.